ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/FullFormTest.kt

```diff
@@ -4,50 +4,27 @@
 
 package io.ktor.client.tests
 
-import io.ktor.application.*
 import io.ktor.client.call.*
-import io.ktor.client.engine.*
 import io.ktor.client.request.*
 import io.ktor.client.response.*
 import io.ktor.client.tests.utils.*
 import io.ktor.http.*
-import io.ktor.request.*
-import io.ktor.response.*
-import io.ktor.routing.*
-import io.ktor.server.engine.*
-import io.ktor.server.jetty.*
+import io.ktor.util.*
+import kotlinx.coroutines.*
+import kotlinx.io.core.*
 import kotlin.test.*
 
-
-@Suppress("KDocMissingDocumentation")
-abstract class FullFormTest(private val factory: HttpClientEngineFactory<*>) : TestWithKtor() {
-    override val server = embeddedServer(Jetty, serverPort) {
-        routing {
-            get("/hello") {
-                assertEquals("Hello, server", call.receive())
-                call.respondText("Hello, client")
-            }
-            post("/hello") {
-                assertEquals("Hello, server", call.receive())
-                call.respondText("Hello, client")
-            }
-            get("/custom") {
-                call.respond(HttpStatusCode(200, "Custom"), "OK")
-            }
-        }
-    }
-
+class FullFormTest : ClientLoader() {
     @Test
-    fun testGet() = clientTest(factory) {
+    fun testGet() = clientTests {
         test { client ->
             val text = client.call {
                 url {
                     protocol = URLProtocol.HTTP
                     host = "127.0.0.1"
-                    port = serverPort
-                    encodedPath = "/hello"
+                    port = 8080
```

Comment:
Why a constant port?

Reply:
We use external test server on the fixed port here